ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/JdbcLedgerDao.scala

```diff
@@ -107,6 +107,17 @@ private class JdbcLedgerDao(
     ()
   }
 
+  private val SQL_UPDATE_EXTERNAL_LEDGER_END = SQL(
+    "update parameters set external_ledger_end = {ExternalLedgerEnd}")
+
+  private def updateExternalLedgerEnd(externalLedgerEnd: LedgerString)(
```

Comment:
Would this force one extra roundtrip to the database for every call that saves the offset?

Reply:
yes it would for each state update.